docs/src/rules/no-inline-comments.md

```diff
@@ -3,6 +3,7 @@ title: no-inline-comments
 rule_type: suggestion
 ---
 
+This rule was **deprecated** in ESLint v9.26.0. Please use [Prettier](https://prettier.io/) instead.
```

Comment:
I don't think that Prettier can replace this rule. ESLint Stylistic has an open [discussion](https://github.com/eslint-stylistic/eslint-stylistic/issues/758) about creating a replacement.

Reply:
Have removed this line as it is generated automatically and we can update it later once we have a new replacement rule?

Reply:
I'm not sure why we'd want to deprecate this rule without a replacement. The rule is already frozen, and #19350 requests moving it to ESLint Stylistic.

Reply:
Yes, we have decided to deprecate this rule only if and when ESLint Stylistic provides a replacement rule.